content/docs/user-guide/settings/az-console.md

```diff
@@ -0,0 +1,108 @@
+---
+title: Access the Settings Registry with Console Commands
+linktitle: Console Commands
+description: Learn how to access the Settings Registry using AZ Console Commands in Open 3D Engine (O3DE).
+weight: 400
+---
+
+The Settings Registry registers a list of commands with [AZ::Console](/docs/user-guide/programming/az-console/) that allow you to add, update, or delete a key within the Settings Registry, and dump keys to the Console output window.
+
+## Console command list
+
+All Settings Registry Console commands are prefixed with `sr_`, which stands for "Settings Registry". This provides easy discovery of settings registry commands.
+
+| <div style="width:150px">Command</div> | Description |
+| :-- | :-- |
+| `sr_regset` \<key> \<value> | Adds or modifies value at the specified key entry in the Settings Registry with the value argument. Multiple keys and values can be specified. |
+| `sr_regremove` \<key> | Removes each key along with its value from the Settings Registry. Multiple keys can be specified. |
+| `sr_regdump` \<key> | Dumps each key value from the Settings Registry to the console output window. If multiple keys are specified, their values are separated by newlines. |
+| `sr_regdumpall` | Dump the entire Settings Registry to the console output window. |
+| `sr_regset_file` \<file path> \<anchor path> | Merges the Settings Registry file at the specified anchor path in the Settings Registry. The anchor path parameter merges the loaded settings under a specific settings object, for example, `"/O3DE/Settings"`. |
+| `sr_dump_origin` \<key> | Outputs the filepath responsible for the latest modification of each key in the Settings Registry.  If the key was was last updated in C++ or Script, then a value of "\<in-memory>" is output.  Multiple keys can be specified.  |
+
+## Examples
+
+### Use `sr_regset` to update or add a value
+
+The following Console command examples update the project root path in the Settings Registry.
+
+| Command | Result |
+| --- | --- |
+| `sr_regset /Amazon/AzCore/Bootstrap/project_path TestProject` | This example sets the "project\_path" field under the "/Amazon/AzCore/Bootstrap" JSON object to "TestProject". |
+| `sr_regset /Amazon/AzCore/Bootstrap/project_path ProjectPath With Spaces` | This example sets the "project\_path" field under the "/Amazon/AzCore/Bootstrap" JSON object to "ProjectPath With Spaces". |
+
+![](/images/user-guide/settings/settings-registry-how-to-console-regset.png)
+
+### Use `sr_regremove` to remove a key value
+
+The following Console command examples remove keys and their values from the Settings Registry.
+
+| Command | Result |
+| --- | --- |
+| `sr_regremove /Amazon/AzCore/Bootstrap/project_path` | This removes the "project\_path" field under the "/Amazon/AzCore/Bootstrap" JSON object. |
```

Comment:
```suggestion
| `sr_regremove /Amazon/AzCore/Bootstrap/project_path` | This removes the "project_path" field under the "/Amazon/AzCore/Bootstrap" JSON object. |
```
You shouldn't need to escape an underscore in the middle of a word.

Reply:
Removing the underscore works.
It is not trying to underline the text.
I'll make the updates